Block/ListProduct.php

```diff
@@ -13,9 +14,43 @@
 use Magento\Catalog\Model\Product;
 use Magento\Framework\Registry;
 use Magento\Catalog\Model\CategoryFactory;
+use Smile\ElasticsuiteCatalog\Model\ResourceModel\Product\Fulltext\CollectionFactory;
 
 class ListProduct extends \Magento\Catalog\Block\Product\ListProduct
 {
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $scopeConfig;
```

Comment:
Please, change all `private` methods and attributes to `protected`, so in case when someone want's to override something in the module, they'll be able to change anything